src/test/java/edu/umich/its/spe/SPEEsbPUTGradeIntegrationTest.java

```diff
@@ -53,11 +53,11 @@ public void tearDown() throws Exception {
 
 	private static Logger M_log = LoggerFactory.getLogger(SPEEsbPUTGradeIntegrationTest.class);
 
-	@Test
-	public void checkPropertiesFile() throws IOException {
-		Properties props = TestingUtils.readTestProperties(speproperties);
-		assertNotNull(props);
-	}
+//	@Test
```

Comment:
Will the checkPropertiesFile() test be included in ASuiteOfMethodTests.java or ASuiteOfESBTests.java?

Instead of commenting out, I would like to remove the checkPropertiesFile() from this file.